crates/bevy_input/src/gamepad.rs

```diff
@@ -3,65 +3,199 @@ use bevy_ecs::event::{EventReader, EventWriter};
 use bevy_ecs::system::{Res, ResMut};
 use bevy_utils::{tracing::info, HashMap, HashSet};
 
+/// A gamepad with an associated `ID`.
+///
+/// ## Usage
```

Comment:
This section should link to `Gamepads`, as the primary way to figure out which gamepads to access.